beacon-chain/p2p/subnets.go

```diff
@@ -109,22 +148,50 @@ func (s *Service) updateSubnetRecordWithMetadata(bitV bitfield.Bitvector64) {
 	})
 }
 
+// Updates the service's discv5 listener record's attestation subnet
+// with a new value for a bitfield of subnets tracked. It also record's
+// the sync committee subnet in the enr. It also updates the node's
+// metadata by increasing the sequence number and the subnets tracked by the node.
+func (s *Service) updateSubnetRecordWithMetadataV2(bitV bitfield.Bitvector64, bitS bitfield.Bitvector4) {
+	entry := enr.WithEntry(attSubnetEnrKey, &bitV)
+	subEntry := enr.WithEntry(syncCommsSubnetEnrKey, &bitS)
+	s.dv5Listener.LocalNode().Set(entry)
+	s.dv5Listener.LocalNode().Set(subEntry)
+	s.metaData = wrapper.WrappedMetadataV1(&pb.MetaDataV1{
+		SeqNumber: s.metaData.SequenceNumber() + 1,
+		Attnets:   bitV,
+		Syncnets:  bitS,
+	})
+}
+
 // Initializes a bitvector of attestation subnets beacon nodes is subscribed to
 // and creates a new ENR entry with its default value.
-func intializeAttSubnets(node *enode.LocalNode) *enode.LocalNode {
+func initializeAttSubnets(node *enode.LocalNode) *enode.LocalNode {
 	bitV := bitfield.NewBitvector64()
 	entry := enr.WithEntry(attSubnetEnrKey, bitV.Bytes())
 	node.Set(entry)
 	return node
 }
 
+// Initializes a bitvector of sync committees subnets beacon nodes is subscribed to
+// and creates a new ENR entry with its default value.
+func initializeSyncCommSubnets(node *enode.LocalNode) *enode.LocalNode {
+	bitV := bitfield.Bitvector4{byte(0x00)}
```

Comment:
You used `bitS` earlier, perhaps use `bitS` for sync related

Reply:
I think `bitV` is better here, we want to denote that this is bitvector. I have changed the cases where I use `bitS` as its ambigious and unclear. Let me know if it works for you.